ansible/pbTestScripts/testJDK.sh

```diff
@@ -1,14 +1,12 @@
 #!/bin/bash
 
-mv /vagrant/pbTestScripts/workspace/build/src/build/linux-x86_64-normal-server-release/images/jdk8* ~
+mv $HOME/openjdk-build/workspace/build/src/build/linux-x86_64-normal-server-release/images/jdk8* $HOME
 export TEST_JDK_HOME=$(find ~ -maxdepth 1 -type d -name "*jdk8u*"|grep -v ".*jre.*")
-cd $HOME && mkdir -p testLocation
-cd testLocation && git clone https://github.com/adoptopenjdk/openjdk-tests
-cd openjdk-tests
-./get.sh -t $HOME/testLocation/openjdk-tests
-cd TestConfig
+mkdir -p $HOME/testLocation
```

Comment:
Any reason not to use `$WORKSPACE` if available? Having jenkins jobs messing around in the user's home directory is not ideal

Reply:
This will be executed on the VMs that are built on the machine. So `$WORKSPACE` will never be set, and messing around in the user's home directory won't affect the machine running the job :-)

Reply:
Fair point